app/src/pages/experiments/DownloadExperimentActionMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { ActionMenu, Item } from "@arizeai/components";
+
+import { Icon, Icons } from "@phoenix/components";
+
+interface DownloadExperimentActionMenuProps {
+  experimentId: string;
+}
+
+export function DownloadExperimentActionMenu({
+  experimentId,
+}: DownloadExperimentActionMenuProps) {
+  return (
+    <div
+      onClick={(e) => {
+        // prevent parent anchor link from being followed
+        e.preventDefault();
+        e.stopPropagation();
+      }}
+    >
```

Comment:
there is a StopPropagation component that does this now https://github.com/Arize-ai/phoenix/blob/main/app/src/components/StopPropagation.tsx